app/src/main/java/ar/com/wolox/android/example/utils/StringUtils.java

```diff
@@ -0,0 +1,26 @@
+package ar.com.wolox.android.example.utils;
+
+import android.support.annotation.Nullable;
+
+/**
+ * Utilities to work with strings;
+ */
+public final class StringUtils {
+
+    static final String EMAIL_REGEX = "[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)" +
+            "*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?";
+
+    /**
+     * Private constructor for utility class;
+     */
+    private StringUtils() {
```

Comment:
It's necessary this constructor if it has no implementation?

Reply:
Checkstyle rule violations were detected. Utility classes should not have a public or default constructor.